blocks/breadcrumb/breadcrumb.css

```
@@ -0,0 +1,61 @@
+.section.breadcrumb-container {
+    background: #F0F5F7;
+    padding: 0;
+    font-size: var(--body-font-size-xs);
+    font-weight: var(--font-weight-medium);
+    line-height: 1;
+    display: none;
+}
+
+.block.breadcrumb {
+    display: flex;
+    list-style: none;
+    padding: 10px 0;
+}
+
+.block.breadcrumb a {
+    color: var(--secondary);
+}
+
+.breadcrumb-item + .breadcrumb-item {
+    padding-left: 8px;
+}
+
+.breadcrumb-item::before {
+    height: 12px;
+    width: 8px;
+    vertical-align: middle;
+    margin-right: 8px;
+    background-image: url('../../icons/angle-right.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+}
+
+.breadcrumb-item + .breadcrumb-item::before {
+    display: inline-block;
+    content: "";
+}
+
+@media (min-width: 992px) {
```

```suggestion
@media (min-width: 62rem) {
```

[As discussed here](https://adobe-dx-support.slack.com/archives/C051MV16P0D/p1685342847436249)